### PR TITLE
Adapt zim-tools to new libzim api.

### PR DESCRIPTION
**Pull request by mgautierfr**
This adapt zim tools to what have been already merged in libzim.

### CODE REVIEW

File: src/zimwriterfs/zimwriterfs.cpp [hunk 1]
```diff
@@ -154,7 +154,7 @@ void usage()
   std::cout << "\t-h, --help\t\tprint this help" << std::endl;
   std::cout << "\t-V, --version\t\tprint the version number" << std::endl;
   std::cout
-      << "\t-m, --minChunkSize\tnumber of bytes per ZIM cluster (default: 2048)"
+      << "\t-m, --minChunkSize\tnumber of kilobytes per ZIM cluster (default: 2048)"
```
Comment (veloman-yunkan) [inline src/zimwriterfs/zimwriterfs.cpp:157]: What about renaming this option?

**Comment by holta** [comment]
I tested this PR.

It works to install zim-tools and zimdump in the context of #242.

File: src/zimwriterfs/zimwriterfs.cpp (hunk 1 above)
Comment (mgautierfr) [reply src/zimwriterfs/zimwriterfs.cpp]: Done.

**Comment by kelson42** [comment]
@mgautierfr Would be great if you could put a word about Mustache in the README.md as well.

**Comment by mgautierfr** [comment]
> Would be great if you could put a word about Mustache in the README.md as well.

It is two different issues.
Here we are fixing compilation dues to a change in libzim. (so zim-tools has to adapt)
Mustache is because we add the dependency in a PR of zim-tools itself.

**Review by veloman-yunkan — CHANGES_REQUESTED** [review]
`long_options` were not updated. I guess you missed it because `minchunksize` is all lowercase there.

**Comment by mgautierfr** [comment]
> long_options were not updated. I guess you missed it because minchunksize is all lowercase there.

Yes. And I've test with `-clusterSize`, which didn't raise a error as it is interpreted as `creator` option :facepalm: 

It is fixed (and already fixup in the commit).

**Review by veloman-yunkan — APPROVED** [review]

**Pull request merged**